backend/docker-compose-opt.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
```

Comment:
Why use version `3` and not `3.8` for example ?

Reply:
Other question: this file is used for what ? `opt` is not explicit I think

Reply:
I used the version `3` as it is enough for the features the file is using.